Restore.cmd

```diff
@@ -35,6 +35,7 @@ if "%RestoreFast%" == "" (
 
 echo Restoring packages: Toolsets
 call "%NugetExe%" restore "%RoslynRoot%build\ToolsetPackages\project.json" %NuGetAdditionalCommandLineArgs% || goto :RestoreFailed
+call "%NugetExe%" restore "%RoslynRoot%build\Toolset.sln" %NuGetAdditionalCommandLineArgs% || goto :RestoreFailed
```

Comment:
I added a "DeployToolsetBouldTask" project that just deploys the contracts. It isn't in any other solution, so it won't be restored unless we restore Toolset.sln